src/Components/Artwork/GridItem.tsx

```diff
@@ -15,7 +15,6 @@ const Placeholder = styled.div`
   background-color: ${color("black10")};
   position: relative;
   width: 100%;
-  overflow: hidden;
```

Comment:
Has this been tested across a variety of different images / pages? There have been issues in the past where images have overflowed, leading to textual content being pushed down in the brick. (I wish I had something more concrete to point to.)